readme.md

```diff
@@ -59,6 +59,15 @@ Default: `true`
 
 Overwrite existing destination file.
 
+##### cwd
+
+Type: `string`\
+Default: `process.cwd()`
+
+Base directory when you use relative path to the source, destination paths.
+
+`cp-file` finds the source file from this path, and copy it to this directory.
```

Comment:
You forgot to update index.d.ts.

Reply:
This sentence is not even correct. That's not how `cwd` works. The first sentence is also not good enough.